network-config-analyzer/nca.py

```diff
@@ -77,32 +110,35 @@ def run_args(args):
     if args.scheme:
         return SchemeRunner(args.scheme, args.output_format, args.file_out).run_scheme()
 
-    ns_list = args.ns_list or 'k8s'
-    pod_list = args.pod_list or 'k8s'
-    base_ns_list = args.base_ns_list or args.ns_list
-    base_pod_list = args.base_pod_list or args.pod_list
     base_np_list = args.base_np_list or 'k8s'
-    output_format = args.output_format or 'txt'
-    cl_execute = CLExecute(base_ns_list, base_pod_list, ns_list, pod_list, output_format, args.file_out, args.pr_url)
+    base_peer_container = PeerContainer(args.base_ns_list or args.ns_list, args.base_pod_list or args.pod_list)
```

Comment:
It might be that both `args.base_ns_list`  and `args.ns_list` are not defined. In this case, the default should be `k8s`.
Same for `args.base_pod_list` and `args.pod_list`.